metrics/examples/prometheus/install/prometheus-rules.yaml

```diff
@@ -48,7 +56,46 @@ spec:
         severity: warning
       annotations:
         summary: 'Kafka offline log directories'
-        description: 'There are {{ $value }} offline log directoris on {{ $labels.kubernetes_pod_name }}'
+        description: 'There are {{ $value }} offline log directories on {{ $labels.kubernetes_pod_name }}'
+    - alert: ScrapeProblem
+      expr: up{job="kubernetes-services",kubernetes_namespace!~"openshift-.+",kubernetes_pod_name=~".+-kafka-[0-9]+"} == 0
+      for: 3m
+      labels:
+        severity: major
+      annotations:
+        summary: 'Prometheus was unable to scrape metrics from {{ $labels.kubernetes_pod_name }}/{{ $labels.instance }}'
+        description: 'Prometheus has not been able to scrape metrics from {{ $labels.kubernetes_pod_name }}/{{ $labels.instance }} for more than 3 minutes'
+    - alert: ClusterOperatorContainerDown
+      expr: count((container_last_seen{container_name="strimzi-cluster-operator"} > (time() - 90))) < 1 or absent(container_last_seen{container_name="strimzi-cluster-operator"})
+      for: 1m
+      labels:
+        severity: major
+      annotations:
+        summary: 'Cluster Operator is down'
+        description: 'Cluster Operator is down for more than 90 seconds'
+    - alert: KafkaBrokerContainerDown
+      expr: absent(container_last_seen{container_name="kafka",kubernetes_pod_name=~".+-kafka-[0-9]+"})
+      for: 3m
+      labels:
+        severity: major
+      annotations:
+        summary: 'Container `kafka` in one of the Kafka pods is down or in CrashLookBackOff status'
+        description: 'Container `kafka` in one of the Kafka pods is down or in CrashLookBackOff status for 3 minutes'
```

Comment:
Can we add the "kubernetes_pod_name" name here so that user knows exactly in which Kafka broker pod, the container kafka is down?

Reply:
It seems this condition is wrong. `absent` needs exact name, not just regex. So we need to copy this alert for each pod or use `or` in the condition. What do you think @ppatierno @scholzj ?

Reply:
No, that would make it completely useless. How can a container be absent?

Reply:
It is "absent" if it is on the CLBO state.

Reply:
if it answers my initial question, I didn't understand :-)